clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -335,6 +335,10 @@ class CIRGenBuilderTy : public CIRBaseBuilderTy {
     }
   }
 
+  bool isCIRIntTypeSize(unsigned N) {
+    return N >= 8 && N <= 64 && llvm::isPowerOf2_32(N);
```

Comment:
Should this be something we ask the dialect instead? Might be easier to keep up to date if we keep adding more integer types (e.g. int128). Perhaps a static method on `cir::IntType` could do the job?

Reply:
@bcardosolopes  done!